### ISSUE TITLE
Warnings about plugin compatibility already showing up in WooCommerce 3.2

### ISSUE THREAD

**Issue by ValeDeOro** [woocommerce/woocommerce#17284]
When you update from WooCommerce 3.2 to 3.2.1, the Status Report shows the following warning messages for extensions: 

```
WooCommerce Stripe Gateway: by WooCommerce – 3.2.3 – Not tested with the active version of WooCommerce WooCommerc
Gravity Forms Product Add-Ons: by Lucas Stark – 3.1.11 – 3.2.2 is available – Not tested with the active version of WooCommerce 
WooCommerce Product Search: by itthinx – 1.10.1 – Not tested with the active version of WooCommerce 
WooCommerce Table Rate Shipping: by WooCommerce – 3.0.4 – Not tested with the active version of WooCommerce 
WooCommerce WooCart Pro: by Festi Team – 2.5 – Not tested with the active version of WooCommerce
```

I expect these warnings when updating to WooCommerce 3.3. in the future, if the plugin headers do not declare compatibility. However, the warnings are showing already. 
This has come up in chat several times.

cc @mattyza and @mikkamp

**Comment by mikkamp** [comment]
On my test site I get the following dialog to popup when I upgrade from 3.2 to 3.2.1

![image](https://user-images.githubusercontent.com/11388669/31759496-80102508-b4a9-11e7-8519-4940a0904ec3.png)

Because it's a test site, I have a large amount of plugins installed without the new header being included. When the list of incompatible plugins is that large there is no scroll bar, so the buttons at the bottom aren't visible. I have to either press `enter` or `esc` to continue there.

**Comment by thomas-zg** [comment]
+1 in Status Report with WC 3.2.1

**Comment by mikejolley** [comment]
The fix for the scrolling is in 3.2.2 already :)

As for the notices, I think they are correct. @claudiosanches seems to agree.

> Not tested with the active version of WooCommerce

Since they define **no** compatible version, it doesn't know if they are tested with 3.2, and in the future won't know if they are tested with 3.3. We need some kind of number to compare against for this to work.

Removing the notice for plugins with no header would pretty much make this function useless as there would be no warning whatsoever. We can put some new dev blog posts out to get these adopted if needed?

cc @mattyza for the woo ones. I assume the headers just roll out as fix releases go out now.

For chats/support, maybe a pre-def would help. It's just an advisory, and our plugins have all been tested with 3.2 so are safe.

thanks!

**Comment by mattyza** [comment]
> cc @mattyza for the woo ones. I assume the headers just roll out as fix releases go out now.

Yep, the extension updates will roll out naturally, as well as deploying all extensions prior to 3.3, to ensure the headers are in place and up to date.
